### PR TITLE
Allowing specific recipes to be "re-installed"

### PR DESCRIPTION
**Pull request by weaverryan**
Hi! 

Very simple: you can now re-install/update just one (or specific) recipes:

```
composer symfony:sync-recipes symfony/console --force
```

That's very helpful, as there are a few specific recipes (this is one of them) that really *do* need to be "updated", at the very least, when upgrading to Symfony 5 (as some important files have evolved over time).

This relates to #575 (will help with upgrading), #518 and should go along with the PRs #562 and #565

Thanks!

### CODE REVIEW

File: src/Command/SyncRecipesCommand.php [hunk 1]
```diff
@@ -58,24 +60,47 @@ protected function execute(InputInterface $input, OutputInterface $output): int
...
             if ($force || !$symfonyLock->has($pkg['name'])) {
                 $packages[] = $pkg['name'];
             }
         }
 
+        $io = $this->getIO();
+
+        if ($targetPackages = $input->getArgument('packages')) {
+            if ($invalidPackages = array_diff($targetPackages, $totalPackages)) {
+                $io->writeError(sprintf('<warning>Invalid package name(s):</warning> %s', implode(', ', $invalidPackages)));
```
Comment (nicolas-grekas) [inline src/Command/SyncRecipesCommand.php]: "invalid"' means "not installed"? maybe we should tell it this way?

File: src/Command/SyncRecipesCommand.php [hunk 2]
```diff
@@ -58,24 +60,47 @@ protected function execute(InputInterface $input, OutputInterface $output): int
...
+                $io->writeError('Re-run the command with <info>--force</info> to re-install the recipes.');
+                $io->writeError('');
+            }
+
+            $packages = array_diff($targetPackages, $packagesRequiringForce);
+        }
+
         if (!$packages) {
+            $io->writeError('No recipes to install.');
+            // print: nothing to install
```
Comment (nicolas-grekas) [inline src/Command/SyncRecipesCommand.php]: should be removed I guess

**Comment by weaverryan** [comment]
This is ready to go

**Comment by fabpot** [comment]
Thank you @weaverryan.

**Pull request merged**